include/seqan3/alignment/pairwise/policy/alignment_matrix_policy.hpp

```diff
@@ -192,10 +191,39 @@ class alignment_matrix_policy
         ++trace_matrix_iter;
     }
 
+    /*!
+     * \brief Converts a sliced position in the alignment matrix to the corresponding position in the original sequence 1.
+     *
+     * \param position The position in the sliced alignment matrix.
+     * \return The corresponding position in the original sequence 1.
+     *
+     * Only changes the position if the sequence was sliced due to band configuration.
+     */
+    constexpr size_t to_original_sequence1_position(size_t position) const noexcept
+    {
+        return position + seq1_slice_offset;
```

Comment:
In theory, it could just be the identity if `!traits_t::is_banded`?

Reply:
Yes. Right now, it would just add 0. However, I guess some extra constexpr wrapper only to get a noop is for this part of the alignment code not performance relevant at all and adds just more complexity.